src/dune

```diff
@@ -0,0 +1,5 @@
+(library
+  (name lwt_binio)
+  (modules lwt_binio)
+  (wrapped false)
```

Comment:
Are the `modules` and `wrapped` stanzas needed here?

Reply:
Contrary to oasis, in dune, wrapped if `true` by default. So it's necessary if you ever add a module to the library, for now you only have one so it's useless but making no harm.

For the modules stanza, it's also useless, but you'll have to add it if you ever add a second library or an executable to this folder. I can remove it if you want but it's making no harm.